src/types.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { DirectusOptions, IDirectus } from '@directus/sdk';
+import { DirectusOptions, IDirectus, TypeMap } from '@directus/sdk';
```

Comment:
Is TypeMap available for SDK 9 and 10 right?

Reply:
Yes, as documented in[DirectusSDK9 Types](https://www.npmjs.com/package/@directus/sdk/v/9.14.2?activeTab=code)